1.9/administration/upgrading.md

```diff
@@ -18,132 +18,75 @@ This document provides instructions for upgrading a DC/OS cluster from version 1
-    ```
-
-1. **Important:** If the type of storage backend for Exhibitor is not set to `exhibitor_storage_backend: static`, add the user `dcos_exhibitor` and give it ownership of the ZooKeeper data directory:
-
-    ```
-    $ sudo useradd --system --home-dir /opt/mesosphere --shell /sbin/nologin -c 'DCOS System User' dcos_exhibitor
-    $ sudo chown -R dcos_exhibitor /var/lib/zookeeper
-    ```
-
-1. Install DC/OS 1.8:
-
-    ```
-    $  sudo bash dcos_install.sh -d master
+    $ curl -O <bootstrap_url>/dcos_node_upgrade.sh
+    $ bash ./dcos_node_upgrade.sh
     ```
 
 1.  Validate the upgrade
 
     - Monitor the Exhibitor UI to confirm that the Master rejoins the ZooKeeper quorum successfully (the status indicator will turn green).  The Exhibitor UI is available at `http://<dcos_master>:8181/`.
     - Verify that `curl http://<dcos_master_private_ip>:5050/metrics/snapshot` has the metric `registrar/log/recovered` with a value of `1`.
-    - Verify that `http://<dcos_master>/mesos` indicates that the upgraded master is running Mesos 1.0.1.
+    - Verify that `http://<dcos_master>/mesos` indicates that the upgraded master is running Mesos 1.2.0.
 
 ### DC/OS Agents
 
-**Important:** When reinstalling on agent nodes, there is a 75 second timeout for the agent to respond to health check pings from the mesos-masters before it is considered lost and its tasks are given up for dead.
+Proceed with upgrading every agent in any order. Agent upgrades can be parallelized with a few caveats. During an upgrade the Mesos agent will go offline briefly (tasks will continue to run) and load on the Master nodes will increase slightly and proportional to the number of Agents rejoining the cluster. For maximum service availability it is recommended to upgrade Agents one at a time and only perform parallel upgrades if the larger upgrade batch size has been well tested with the same workload.
 
 ### On all DC/OS Agents:
 
-1.  Download The dcos_install.sh Script
-
-    ```
-    $ curl -O <bootstrap_url>/dcos_install.sh
-    ```
-
-1.  Uninstall pkgpanda
-
-    ```
-    $ sudo -i /opt/mesosphere/bin/pkgpanda uninstall
-    ```
-
-1.  Remove The DC/OS 1.8 Data Directory and add `mesos-resources`
-
-    ```
-    $ sudo rm -rf /opt/mesosphere /etc/mesosphere
-    ```
-
-1.  The DC/OS installer discovers and combines your agent's configuration parameters into a file named `mesos-resources`. As a part of this process the available disk space of each node is calculated. If you have not made explicit disk size reservations, you must create a placeholder for the disk reservation file. This prevents the installer from building a new disk reservation file that might conflict with your stored agent metadata:
+1.  Download and run the `dcos_node_upgrade.sh` script:
 
     ```
-    $ sudo mkdir -p /var/lib/dcos
-    $ sudo touch /var/lib/dcos/mesos-resources
+    $ curl -O <bootstrap_url>/dcos_node_upgrade.sh
+    $ bash ./dcos_node_upgrade.sh
```

Comment:
`$ sudo bash dcos_node_upgrade.sh` to match the invocation for the install script documented in the advanced install guide.